src/styles/blocks/product-card.scss

```
@@ -0,0 +1,103 @@
+.product-card {
+  margin-bottom: 46px;
+  height: $card-height;
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid $color-white;
+  border-radius: 5px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: $card-margin-rigth;
+  }
+
+  &__container {
+    padding: 32px 16px 0;
+  }
+
+  &__img {
+    padding: 0 3px;
+    margin-bottom: 36px;
+  }
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-cold-oceano;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-grey;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 20px;
+    height: 20px;
+  }
+
+  &__stars-text {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-cold-oceano;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      color: $color-grey;
+    }
+  }
```

Don't make element of another element, use nesting only for BEM block and it's elements
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    line-height: 18px;
    margin-bottom: 16px;
  }
  
  &-text {
      font-weight: 400;
      font-size: 12px;
      color: $color-grey;
  }
```